domain-models-runtime/src/test/java/org/folio/rest/persist/PostgresClientIT.java

```diff
@@ -2219,51 +2222,447 @@ public void selectDistinctOn(TestContext context) throws IOException {
   }
 
   @Test
-  public void streamGet(TestContext context) throws IOException {
+  public void streamGetLegacy(TestContext context) throws IOException {
     AtomicInteger objectCount = new AtomicInteger();
     Async async = context.async();
 
     final String tableDefiniton = "id UUID PRIMARY KEY , jsonb JSONB NOT NULL, distinct_test_field TEXT";
 
     createTableWithPoLines(context, MOCK_POLINES_TABLE, tableDefiniton);
     postgresClient.streamGet(MOCK_POLINES_TABLE, new Object(), "jsonb", null, false, null,
-        streamHandler -> objectCount.incrementAndGet(), asyncResult -> {
-          context.assertEquals(6, objectCount.get());
+      streamHandler -> objectCount.incrementAndGet(), asyncResult -> {
+        context.assertTrue(asyncResult.succeeded());
+        context.assertEquals(6, objectCount.get());
+        async.complete();
+      });
+    async.awaitSuccess();
```

Comment:
Can you remove `Async async` and use `context.asyncAssertSuccess` instead? Example:

    @Test
    public void streamGetLegacy(TestContext context) throws IOException {
      AtomicInteger objectCount = new AtomicInteger();

      final String tableDefiniton = "id UUID PRIMARY KEY , jsonb JSONB NOT NULL, distinct_test_field TEXT";

      createTableWithPoLines(context, MOCK_POLINES_TABLE, tableDefiniton);
      postgresClient.streamGet(MOCK_POLINES_TABLE, new Object(), "jsonb", null, false, null,
            streamHandler -> objectCount.incrementAndGet(), context.asyncAssertSuccess(asyncResult -> {
            context.assertEquals(6, objectCount.get());
          }));
    }